orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/SaltTelemetryOrchestrator.java

```diff
@@ -429,4 +451,44 @@ private boolean isAnyFilecollectorConfigKeyNotEmpty(Map<String, Object> filecoll
                     return CollectionUtils.isNotEmpty(hostSet);
                 });
     }
+
+    private void executeVmPreFlightCheck(SaltConnector sc, Target<String> targets, Set<Node> nodes, Map<String, Object> parameters)
+            throws CloudbreakOrchestratorFailedException {
+        String command = String.format("%s/%s master-check", REMOTE_SCRIPTS_LOCATION, SCRIPTS_TO_UPLOAD[0]);
```

Comment:
I know it's copy paste but this looks fragile, if someone changes the order of the array.